### ISSUE TITLE
Bug: Fee not adding when EX permit is split by street (ON PROD🎉🎉)

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-data-tech#7352]
### More info to come

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder I have sent this over to the DevOps CM Team to deploy to **_DEV_** under **SCTASK0173015**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan 
I sent this to the DevOps CM Team to be deployed to **_DEV_** under **SCTASK0174530**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan I have sent this to the DevOps M Team to Deploy to **_DEV_** under **SCTASK0177493**

**Comment by Nadin-Nader** [comment]
**Status:**
Waiting on the DevOps CM Team to Deploy to **_DEV_** under **SCTASK0177493**

**Comment by Nadin-Nader** [comment]
**Status:**
This is having a merge conflict and needs to be addressed when Sree gets back 11/4. Martin has given his advice on what to do in order to mitigate the issue, see below:

> Heres what we can do let me know if this makes sense. 
> 
> Grab a copy from dev. Commit that. 
> 
> Then do your change changing from autobill to calling procedure and commit that . 
> Then we can promote it on up.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan whats the status of the conversation with Martin?

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan I think this was the response you needed. I noticed that the email to you may be the old one. I will paste the email response to you, below:

> 
> From: Aponte, Martin <Martin.Aponte@austintexas.gov> 
> Sent: Wednesday, November 3, 2021 9:57 AM
> To: Nader, Nadin <Nadin.Nader@austintexas.gov>; Blaze Kannan, Sreelatha - Vendor <ex-Sreelatha.BlazeKannan@austintexas.gov>; Shirai, George <George.Shirai@austintexas.gov>
> Subject: RE: SCTASK0177493 bfc0ff295c19 - merge conflicts 
> 
> Not as far as I’m aware It’s a fee change so it may be high priority but its lasted this long don’t think another day will hurt. 
> 
> From: Nader, Nadin <Nadin.Nader@austintexas.gov> 
> Sent: Wednesday, November 3, 2021 9:45 AM
> To: Aponte, Martin <Martin.Aponte@austintexas.gov>; Blaze Kannan, Sreelatha - Vendor <ex-Sreelatha.BlazeKannan@austintexas.gov>; Shirai, George <George.Shirai@austintexas.gov>
> Subject: RE: SCTASK0177493 bfc0ff295c19 - merge conflicts 
> 
> Hi Martin,
> Hope you are having a good morning. Sree is out today. I will make sure she follows up after she tries your advice. My question is, is there any issue that can come up from it being done tomorrow?
> 
> From: Aponte, Martin <Martin.Aponte@austintexas.gov> 
> Sent: Wednesday, November 3, 2021 8:09 AM
> To: Blaze Kannan, Sreelatha - Vendor <ex-Sreelatha.BlazeKannan@austintexas.gov>; Shirai, George <George.Shirai@austintexas.gov>; Nader, Nadin <Nadin.Nader@austintexas.gov>
> Subject: RE: SCTASK0177493 bfc0ff295c19 - merge conflicts 
> 
> Hello,
> 
> Not sure why it showing as new I see a few dummy commits but I can easily track it. 
> 
> Heres what we can do let me know if this makes sense. 
> 
> Grab a copy from dev. Commit that. 
> 
> Then do your change changing from autobill to calling procedure and commit that . 
> Then we can promote it on up.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Earlier changes were not working as expected, so made few changes to the code and sent the commit. As it is portal changes can't be tested in Localhost. Please approve and create task id for me. Here is the commit id: 0c074f9563ef

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been sent to the DevOps CM team to be deployed to **_DEV_** under **SCTASK0180603**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been deployed to **_DEV_** please UAT

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan Can you do the UAT on DEV for this as soon as you can. the stakeholders dont really have access to test on DEV.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan This has been sent to the DevOps CM Team to be deployed to **_DEV_** under **SCTASK0183554**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been deployed to DEV, Please UAT

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Debugging is in progress. But yet to identify the issue. tried various methods to make it work. due to the portal not being able to understand how it is failing. as it runs well when I try to run from the database.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - I need Clarification on this before I submit my DEV DP on this.

Only for Private Development, it is inserting fee in the Child folder as per the current flow, do you want me to do the same, or do we need to insert the fee for all subcodes except franchise utility and bill according to our new fee logic (based on emergency & Secondary info's)

**Comment by TracyLinder** [comment]
@Sreelatha-BlazeKannan for all EX (except Franchise), the fee needs to add to the child folder(s). So you are correct.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - DEV DP Submitted; Please approve and create task id for the same. 
Here is the commit ID: db8151e36c4c

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Tested in DEV, the fee is getting added to Child folder but not getting auto paid for COS subtypes; Added changes for this and submitted another DEV DP. Please approve and create task id for me. Commit ID: dab91a49bf3e

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this was sent to the DevOps CM Team for deployment to **_DEV_** Under **SCTASK0187978**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder this has been deployed to **DEV**

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Tested in DEV, now it splits, autobills and COS auto pays also; however for Private development and Project Connect subtypes Child folder not getting autobilled like Parent folder. To fix that issue submitted a commit today. Here is the commit ID:e8ab4b13411a. Please approve and create task id for me.

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan I have sent this to the DevOps CM Team to deploy to **_DEV_** under **SCTASK0188569**

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder  - Tested DEV Changes and all working as expected, #7352 TEST DP for your reference. Please approve the below commits and create task id for me. These needs to be promoted in the below order.

Promoting Order 1 Commit ID: db8151e36c4c
Promoting Order 2 Commit ID: dab91a49bf3e
Promoting Order 3 Commit ID: e8ab4b13411a

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this ahs been sent to the DevOps CM Team to deploy to **_TEST_** under **SCTASK0188886**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been deployed to **TEST** Please UAT

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Completed testing in TEST Environment and it is working as expected. Please let the stake holders know and this can be tested and ready for TEST UAT and PROD Move.

**Comment by Nadin-Nader** [comment]
**Status:**
Waiting on Stakeholders to UAT

**Comment by Nadin-Nader** [comment]
**Status:**
Waiting on UAT from Stakeholders but that is on hold due to the bug fixing to the Portal Integration with City Base. This can be UATd early next week.

**Comment by Sreelatha-BlazeKannan** [comment]
@Nadin-Nader , @TracyLinder - Upon UAT Approval, Submitted #7352 PROD DP for your reference. Please approve the below commits and create task id for me. These needs to be promoted in the below order.

Promoting Order 1 Commit ID: "db8151e36c4c"
Promoting Order 2 Commit ID: "dab91a49bf3e"
Promoting Order 3 Commit ID: "e8ab4b13411a"

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan this has been sent to the DevOps CM Team to be deployed to **_PROD_** under **SCTASK0193397**

**Comment by Nadin-Nader** [comment]
@Sreelatha-BlazeKannan @TracyLinder this has been deployed to PROD. I will let the stakeholders know and close it either today or next week (if they want time to make sure it works on PROD)

**Comment by Nadin-Nader** [comment]
**Status:**
ON PROD but waiting until Friday to close

![image.png](https://images.zenhubusercontent.com/5e8dfbfabf171441c3a5684e/926871c3-fc8f-4285-b41d-8a1dbab9bbd3)

**Comment by TracyLinder** [comment]
I just did UAT on PROD and the fees were added to both permits. I'm closing this. 
- Test cases: 22-012662 & 665
